### PR TITLE
modify build dir of copilot_proxy

### PR DESCRIPTION
**Pull request by luanshaotong**
After setup, models directory can take up a lot of space.
It will take long time to build copilot_proxy image for "Sending build context to Docker daemon".
This commit changes build directory to `./copilot_proxy`, to avoid sending build context in root directory.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -23,8 +23,8 @@ services:
     # command: python3 -m flask run --host=0.0.0.0 --port=5000
     # For local build
     build:
-      context: .
-      dockerfile: copilot_proxy/Dockerfile
+      context: ./copilot_proxy
```
Comment (leemgs) [inline docker-compose.yaml:26]: It looks good to me at least. :)

File: copilot_proxy/Dockerfile [hunk 2]
```diff
@@ -2,11 +2,11 @@ FROM python:3.10-slim-buster
 
 WORKDIR /python-docker
 
-COPY copilot_proxy/requirements.txt requirements.txt
+COPY requirements.txt requirements.txt
```
Comment (leemgs) [inline copilot_proxy/Dockerfile:5]: It seems that the existing statement is better than this trial for readability and maintenance. :(

Comment (luanshaotong) [reply copilot_proxy/Dockerfile:5]: Maybe this trial is in the same format as the converter. :)

**Comment by Frederisk** [comment]
This problem is also caused by #49, I think it's a good idea to rollback it and do the same PR after a while. At least it might be better for posting this image on DockerHub.

**Comment by moyix** [comment]
Is the issue here mainly with the `mymodels` directory? If so, maybe a cleaner fix would be to just add that to our existing .dockerignore file?

**Comment by Frederisk** [comment]
Hmm, I mention #49 because: in 6b69437b7e95d5d7c3efb88bdda0d5a8248bd95c, `context` has moved from the original location to the current folder.

**Comment by fdegier** [comment]
A boring solution could be to move the Dockerfiles to the root, move models to `converter`, by having the Dockerfiles in the root, we can also have dockerignores per Dockerfile while keeping the convenience of building from the root.

e.g. 

```
api.Dockerfile
api.dockerignore

converter.Dockerfile
converter.dockerignore
```

**Comment by fdegier** [comment]
Thanks for your contribution @luanshaotong !

**Pull request merged**